docs/install.rst

```diff
@@ -81,18 +81,12 @@ activated, so running the tests is as simple as::
 
 .. note::
 
-    If your local tests run fine, but when you submit a pull-request our Jenkins
+    If your local tests run fine, but when you submit a pull-request our CircleCI
     (continuous integration service) instance tells you the tests failed, it could
```

Comment:
We don't have a circleci instance. suggest rewording: "[...] pull request the tests fail in [CircleCI](https://circleci.com/gh/mozilla/bedrock), it could [...]"